docs/TestingFabricProxyModel.rst

```diff
@@ -92,6 +94,57 @@ To run Fabric with Hyperledger Avalon, follow these steps:
 
        ./scripts/start_fabric.sh -d
 
+Setup Hyperledger Avalon Fabric Proxy Model on multiple machine
+----------------------------------------------------------------
+Multinode Hyperledger Avalon fabric proxy model setup requires certain modifications to existing
+docker files. For example to setup the fabric blockchain network on a separate machine and avalon on a separate machine
```

Comment:
Change avalon to Avalon here on line 100 and on line 105, 114, 126.

Reply:
Done